packages/lesswrong/server/callbacks/commentCallbackFunctions.tsx

```diff
@@ -92,8 +92,17 @@ const utils = {
   }) => {
     const rateLimit = await rateLimitDateWhenUserNextAbleToComment(user, comment.postId ?? null, context);
     if (rateLimit) {
-      const {nextEligible, rateLimitType:_} = rateLimit;
+      const {nextEligible, rateLimitType, rateLimitName} = rateLimit;
       if (nextEligible > new Date()) {
+        captureEvent("commentBlockedDueToRateLimit", {
+          rateLimitType,
+          rateLimitName,
+          userId: user._id,
+          userKarma: user.karma,
+          postId: comment.postId,
+          nextEligible: nextEligible.toISOString()
+        });
```

Comment:
@Will-Howard I believe the reason we have the separate `trackCommentRateLimitHit()` function is because, when you are at the rate limit, you cannot submit more comments. So I don't know if this new event will ever trigger?

For example, with a newly setup account I published three comments. After that the "Comment" button is disabled.

<img width="670" height="617" alt="Screenshot 2025-08-05 at 5 13 19 PM" src="https://github.com/user-attachments/assets/a591df42-1ee1-4d9d-8a19-e21dec343dec" />

Reply:
Ah that is true, thank you for pointing it out. We do still see some on prod, which I think is due to the bug where trying to save a draft can be blocked by a rate limit.

I will leave the event in for now as it will be useful to see how it changes when we fix that bug, but I'll then remove it.